ml-proto/test/labels.wast

```diff
@@ -99,6 +99,13 @@
     )
   )
 
+  (func $redefinition (result i32)
+    (block $l1
+      (i32.add (block $l1
+		 (i32.const 2))
```

Comment:
There is no style guide for the formatting of s-exp. Putting trailing braces on separate lines is what I see beginners do and once people have a little experience they prefer the increased information density of placing them on one line - you will not find experienced lisp codes putting closing braces on separate lines as a style rule (there are some exceptions, but not for something so simple as above).